frontend/src/App.tsx

```diff
@@ -4,6 +4,7 @@ import { userData } from "./requests/types";
 import { UserInterface } from "./components/UserInterface";
 import { Routes, Route, BrowserRouter as Router } from "react-router-dom";
 import Home from "./pages/Home";
+import Onboarding from "./pages/Onboarding";
```

Comment:
All pages are named using lowercase. An error may occur but will go away once you merge prod